### DISCUSSION TITLE
Cosmos DB: Partial document update support?

### DISCUSSION THREAD

**Discussion by jacksorjacksor** [Azure/data-api-builder#1991] [Q&A] [answered]
Hi there!

Does DAB with Cosmos DB support [Partial document update](https://learn.microsoft.com/en-us/azure/cosmos-db/partial-document-update)? We're looking to update an item, but we seem to need to send the entire item, rather than just the fields which need to changed.

i.e. if I have an item with "first":"a" and "second":"b" and we're looking to just change "first":"c", sending this update would remove "second" completely. We'd need to send an item containing "first":"c", "second":"b".

Cheers!

Rich

**Comment by sajeetharan** [comment]
Thanks's for the feedback, we are working on it.

**Comment by sajeetharan** [comment, marked as answer]
Hi @jacksorjacksor jacksorjacksor We've released this support with the version https://github.com/Azure/data-api-builder/releases/tag/v1.1.7. Hope you already started using it. let us know if you have any feedback!

**Comment by jacksorjacksor** [comment]
Hi @sajeetharan  - ah great news! We're no longer using CosmosDB for this project so I wouldn't be able to comment. I'll close this Discussion for good housekeeping, and thanks to you and the team for your work on it!
